docs/Running-behind-h2o.md

```diff
@@ -0,0 +1,183 @@
+<!--
+title: "Running Netdata behind H2O"
+custom_edit_url: https://github.com/netdata/netdata/edit/master/docs/Running-behind-h2o.md
+-->
+
+# Running Netdata behind H2O
+
+[H2O](https://h2o.examp1e.net/) is a new generation HTTP server that provides quicker response to users with less CPU utilization when compared to older generation of web servers.
+
+It is notable for having much simpler configuration than many popular HTTP servers, low resource requirements, and integrated native support for many things that other HTTP servers may need special setup to use.
+
+## Why H2O
+
+-   Sane configuration defaults mean that typical configurations are very minimalistic and easy to work with.
+
+-   Native support for HTTP/2 provides improved performance when accessing the Netdata dashboard remotely.
+
+-   Password protect access to the Netdata dashboard without requiring Netdata Cloud.
+
+## H2O configuration file.
+
+On most systems, the H2O configuration is found under `/etc/h2o`. H2O uses [YAML 1.1](https://yaml.org/spec/1.1/), with a few special extensions, for it’s configuration files, with the main configuration file being `/etc/h2o/h2o.conf`. 
+
+You can edit the H2O configuration file with Nano, Vim or any other text editors you are comfortable with.
+
+After making changes to the configuration files:
+
+-   Test the configuration with `h2o -m test -c /etc/h2o/h2o.conf`
+
+-   Restart H2O to apply tha changes with `/etc/init.d/h2o restart` or `service h2o restart`
+
+## Ways to access Netdata via H2O
```

Comment:
In the ## Ways to access Netdata via H2O section, would it be overkill to offer a list of the ways and when they're recommended, and have each item link to the corresponding ### section? 

I made a few commits in the file that were just stylistic/grammar. This is the only other comment so I'll leave this to your discretion!

Reply:
I agree that this would be a good change, but I would argue that it should be made across all of the documents on running behind a proxy, not just this one, so I think it’s probably better done as a separate PR changing all the files.